sources/libs/fs/fs.h

```diff
@@ -8,4 +8,5 @@ typedef enum
     FS_INVALID_INODE,
     FS_INVALID_PATH,
     FS_INVALID_FS,
+    FS_NO_FREE_BLOCK
 } FsResult;
```

Comment:
The Fs subsystem should use IoResult, making interop between our IO and FS subsystem easier.